proto/internal/temporal/server/api/deployment/v1/message.proto

```diff
@@ -151,18 +233,58 @@ message CheckWorkerDeploymentUserDataPropagationRequest {
 message SetCurrentVersionArgs {
     string identity = 1;
     string version = 2;
-    string request_id = 4;
 }
 
 // used as Worker Deployment update response:
 message SetCurrentVersionResponse {
+    string previous_version = 1;
+}
+
+// used as Worker Deployment workflow update input:
+message DeleteVersionArgs {
+    string identity = 1;
+    string version = 2;
+}
+
+// used as Worker Deployment Activity input:
+message DeleteVersionActivityArgs {
+    string identity = 1;
+    string deployment_name = 2;
+    string version = 3;
+    string request_id = 4;
+}
+
+// used as Worker Deployment Activity input:
+message CheckTaskQueuesHaveNoPollersActivityArgs {
+    repeated temporal.api.taskqueue.v1.TaskQueue task_queues = 1;
+    string build_id = 3;
```

Comment:
Should this be `version`? (also number 2 is unused)

Reply:
this is build id because this activity calls DescribeTaskQueueEnhanced which takes a build id

Reply:
numbering also fixed in latest PR